CHANGELOG.md

```diff
@@ -105,6 +105,20 @@ ServiceDescription("First", "Second", "DontCare") myServiceDescription2;
 ServiceDescription("Foo", "Bar", "Baz") myServiceDescription3;
 ```
 
+The service-related methods have been move from `PoshRuntime` to a separate class (TBD):
+
+```cpp
+// before
+poshRuntime.offerService(myServiceDescription);
+poshRuntime.stopOfferService(myServiceDescription);
+poshRuntime.findService(myServiceDescription);
+
+// after
+discoveryInfo.offerService(myServiceDescription);
+discoveryInfo.stopOfferService(myServiceDescription);
+discoveryInfo.findService("ServiceA", Wildcard_t);
```

Comment:
To make this comparable, the `findService` from the `before` part needs to have a service description with `AnyInstance`. Could you please adjust it?

Reply:
@MatthiasKillat Indeed, adapt the example accordingly. Our coding guidelines are still a bit of a mishmash. Do you maybe want to raise this topic in the next developer meetup?